apps/tehwolfde/src/assets/styles/_tokens.scss

```
@@ -0,0 +1,82 @@
+// Design tokens — single source of truth for the tehw0lf brand.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)